Added Few DL Profs in Canada

quettabit opened the pull request.

quettabit committed:
Added Few DL Profs in Canada

emeryberger commented on faculty-affiliations.csv:
```diff
@@ -3357,6 +3359,7 @@ Gregory Dudek , McGill University
 Hamed Hatami , McGill University
 Jackie Chi Kit Cheung , McGill University
 Jérôme Waldispühl , McGill University
+Joelle Pineau , McGill University
```
Does not meet inclusion criteria as will be full-time employed elsewhere.

quettabit replied:
Fine ..

emeryberger commented on faculty-affiliations.csv:
```diff
@@ -8798,9 +8802,11 @@ Gena Hahn , University of Montreal
 Gilles Brassard , University of Montreal
 Guy Lapalme , University of Montreal
 Houari A. Sahraoui , University of Montreal
+Ioannis (Yannis) Mitliagkas , University of Montreal
```
This is not the correct DBLP entry. Please ensure that all entries correspond to the faculty name(s) given at [DBLP](http://dblp.org).

quettabit replied:
Sure ..

emeryberger reviewed (changes requested):
Please check over all faculty names and correct commented entries. Thanks.

quettabit committed:
Made the following changes:
1. Removed Joelle Pineau
2. Removed Jian Tang because of conflicts in dblp. Have mailed dblp guys
3. Added other name in dblp for Xiaodan Zhu

quettabit commented:
@emeryberger have a look at the latest commit message. Acted on the requested changes. Thanks.

emeryberger commented:
Thanks!
